src/frontend/src/app/scheduled-process/scheduled-process.component.ts

```diff
@@ -137,12 +138,36 @@ export class ScheduledProcessComponent implements OnInit {
     this.schedulesService.getTasks("RUNNING").
       subscribe(
       data => {
+        if(data.error){
+           this.alertService.error(data.error)
+        }
         this.tasksData = data.tasks;
         console.log("Running tasks ", this.tasksData)
       },
       error => { console.log("error", error) })
   }
 
+  public cancelRunninTask(id){
+    console.log("ID", id);
+     this.schedulesService.cancelTask(id).
+      subscribe(
+      data => {
+        if(data.error){
+           this.alertService.error(data.error)
+        }
+        if(data.message){
+          this.selectedTaskType = 'Running';
+          setTimeout(()=>
+          {this.getRunningTasks()},300);
+          this.alertService.success(data.message)
+          this.getRunningTasks();
+        }
```

Comment:
logic should be:
```
this.alertService.success(data.message)
```
wait for 5 seconds and show loading indicator mask on tasks grid

```
if (this.selectedTaskType == 'Running')
 this.getRunningTasks();
else 
  this.getLatestTasks()
```